assets/onboarding/data/reducer.js

```diff
@@ -0,0 +1,76 @@
+import {
+	FETCH_USAGE_TRACKING,
+	SUBMIT_USAGE_TRACKING,
+	ERROR_USAGE_TRACKING,
+	SET_USAGE_TRACKING,
+} from './constants';
+
+const DEFAULT_STATE = {
+	welcome: {
+		isFetching: false,
+		isSubmitting: false,
+		error: false,
+		data: {
+			usageTracking: false,
+		},
+	},
+};
+
+/**
+ * Setup wizard reducer.
+ *
+ * @param {Object}         state  Current state.
+ * @param {{type: string}} action Action to update the state.
+ *
+ * @return {Object} State updated.
+ */
+export default ( state = DEFAULT_STATE, action ) => {
+	switch ( action.type ) {
+		case FETCH_USAGE_TRACKING:
+			return {
+				...state,
+				welcome: {
+					...state.welcome,
+					isFetching: true,
+				},
+			};
+
+		case SUBMIT_USAGE_TRACKING:
+			return {
+				...state,
+				welcome: {
+					...state.welcome,
+					isSubmitting: true,
+				},
+			};
+
+		case ERROR_USAGE_TRACKING:
+			return {
```

Comment:
We are not treating error while saving. I added it to the store, so if we desire to show some error message or something like that, we can.